pkg/guestagent/docker/events.go

```diff
@@ -0,0 +1,228 @@
+package docker
+
+import (
+	"context"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/events"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+	"github.com/docker/go-connections/nat"
+	"github.com/lima-vm/lima/pkg/guestagent/api"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	startEvent = "start"
+	stopEvent  = "stop"
+	// die event is a confirmation of kill event.
+	dieEvent = "die"
```

Comment:
Aren't these the same as `events.ActionStart`, `events.ActionStop`, and `events.ActioneDie`?